src/test/java/kitchenpos/common/ServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package kitchenpos.common;
+
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.BEFORE_TEST_METHOD;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@Sql(value = "classpath:test_truncate_table.sql", executionPhase = BEFORE_TEST_METHOD)
```

Comment:
`@Sql`의 `executionPhase`는 default값이 `BEFORE_TEST_METHOD`라 제거해도 괜찮을 것 같아요!
```suggestion
@Sql("/test_truncate_table.sql")
```

Reply:
몰랐던 부분이네요 감사합니다!